### PR TITLE
HTTPCORE-693 - Add HttpHeaders headers.

### PR DESCRIPTION
**Pull request by arturobernalg**
Add following header:

- Accept-Patch : https://tools.ietf.org/html/rfc5789#section-3.1
- HTTP2-Settings 
- A_IM : https://datatracker.ietf.org/doc/html/rfc3229
- Alt-Svc
- P3P
- Preference-Applied: https://datatracker.ietf.org/doc/html/rfc7240#page-7
- Strict-Transport-Security

### CODE REVIEW

File: httpcore5/src/main/java/org/apache/hc/core5/http/HttpHeaders.java [hunk 1]
```diff
@@ -131,6 +143,10 @@ private HttpHeaders() {
     public static final String FROM = "From";
 
     public static final String HOST = "Host";
+    /**
+     * The HTTP {@code HTTP2-Settings} header field name.
+     */
+    public static final String HTTP2_SETTINGS = " HTTP2-Settings";
```
Comment (garydgregory) [inline httpcore5/src/main/java/org/apache/hc/core5/http/HttpHeaders.java:149]: Is there really a space?

File: httpcore5/src/main/java/org/apache/hc/core5/http/HttpHeaders.java [hunk 2]
```diff
@@ -162,12 +178,20 @@ private HttpHeaders() {
     public static final String OVERWRITE = "Overwrite";
 
     public static final String PRAGMA = "Pragma";
+    /**
```
Comment (garydgregory) [inline httpcore5/src/main/java/org/apache/hc/core5/http/HttpHeaders.java:181]: I would add a blank line before a Javadoc comment.

**Review by garydgregory — CHANGES_REQUESTED** [review]

File: httpcore5/src/main/java/org/apache/hc/core5/http/HttpHeaders.java (hunk 1 above)
Comment (arturobernalg) [reply httpcore5/src/main/java/org/apache/hc/core5/http/HttpHeaders.java]: Hi @garydgregory 
Fixed.
TY

**Comment by ok2c** [comment]
@arturobernalg No one seems to be willing to approve this PR. I support it should be closed.

**Comment by arturobernalg** [comment]
> @arturobernalg No one seems to be willing to approve this PR. I support it should be closed.

it seem so. Close the PR

**Pull request closed without merging**